### PR TITLE
Implemented 'gulp serverTestRunner' which lints/tests server files on update.

### PR DESCRIPTION
**Pull request by recursion**
This is basically what running `gulp` does except there is no browser-sync, no watching client-files, not even a server running. 

just run `gulp serverTestRunner` and gulp will watch all server files for changes, then lint and test (with nyan cat reporting)

### CODE REVIEW

File: server/models/Order.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 var bookshelf = require('../utils/bookshelf');
 var uuid = require('node-uuid');
 
+
```
Comment (mnaamani) [inline server/models/Order.js]: ? :)

**Comment by mnaamani** [comment]
I tried it out.. and when it starts it doesn't run the tests until a file actually changes.
After running the tests successfully, gulp exits.
Can we have it so it keeps running.. ?

**Comment by recursion** [comment]
Oh! man nothing gets past you!
Nice catch! Ill see what I can do about that asap.

**Comment by mnaamani** [comment]
great addition to our dev workflow.. merging

**Pull request merged**